pinot-perf/src/main/java/com/linkedin/pinot/perf/QueryRunner.java

```diff
@@ -19,58 +19,122 @@
 import java.io.File;
 import java.io.FileInputStream;
 import java.io.FileReader;
+import java.lang.reflect.Field;
 import java.util.List;
 import java.util.Random;
+import java.util.concurrent.ConcurrentLinkedQueue;
 import java.util.concurrent.ExecutorService;
 import java.util.concurrent.Executors;
 import java.util.concurrent.atomic.AtomicInteger;
 import java.util.concurrent.atomic.AtomicLong;
 import org.apache.commons.io.IOUtils;
 import org.json.JSONObject;
+import org.kohsuke.args4j.CmdLineParser;
+import org.kohsuke.args4j.Option;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+
 /**
  * Simple code to run queries against a server
  * USAGE: QueryRunner QueryFile ({BrokerHost} {BrokerPort} <single-threaded/multi-threaded>)
  */
 public class QueryRunner {
-  private QueryRunner() {
-  }
+  @Option(name = "-queryFile", required = true, usage = "query file path")
+  private String _queryFile;
+  @Option(name = "-mode", required = true, usage = "query runner mode (singleThread|multiThreads|targetQPS)")
+  private String _mode;
+  @Option(name = "-numThreads", required = false, usage = "number of threads for multiThread mode and targetQPS mode")
+  private int _numThreads;
+  @Option(name = "-startQPS", required = false, usage = "start QPS for targetQPS mode")
+  private double _startQPS;
+  @Option(name = "-deltaQPS", required = false, usage = "delta QPS for targetQPS mode")
```

Comment:
Not a requirement for this commit, but you may want to consider adding checks like mutually exclusive arguments, dependent arguments etc.